src/components/movie-view/movie-view.jsx

```diff
@@ -8,12 +10,47 @@ export const MovieView = ({ movie, onBackClick }) => {
           <span>Title: </span>
           <span>{movie.title}</span>
         </div>
+        <div>
+          <span>Description: </span>
+          <span>{movie.description}</span>
+        </div>
+        <div>
+          <span>Genre: </span>
```

Comment:
You will need to make the following changes to your `movie-view` to get your app to work as expected.

- Remove the commented portion of code.

- For both Genre and Director use their correct attributes case for the underlined portion.

![Screenshot 2024-05-10 082303](https://github.com/behailu-2023/myFlix-client/assets/37598493/b260040a-f01e-4b07-9f39-c490a891c687)